apps/ledger-live-desktop/src/renderer/components/TransactionConfirm/ConfirmFooter.tsx

```diff
@@ -23,12 +26,14 @@ if (getEnv("PLAYWRIGHT_RUN")) {
 type Props = {
   footer: React.ReactNode | undefined;
   manifestId?: string | null;
+  transaction?: Transaction | null;
   manifestName?: string | null;
 };
 
-const ConfirmFooter = ({ footer, manifestId, manifestName }: Props) => {
+const ConfirmFooter = ({ footer, transaction, manifestId, manifestName }: Props) => {
   if (!manifestId) return;
-  const termsOfUseUrl = termsOfUse.get(manifestId);
+  const appNameByAddr = transaction?.recipient === uniwapUniversalRouterAddr ? "Uniswap" : null;
```

Comment:
Move Uniswap data to some common library, this info is not specific to desktop.

dexProviders = [
{ name: "Uniswap",
addr:"0x3fC91A3afd70395Cd496C647d5a6CC9D4B2b7FAD" ]

Reply:
Done :)

Reply:
```  
export const dexProvidersContractAddress: { [key: string]: string } = {
  "0x3fC91A3afd70395Cd496C647d5a6CC9D4B2b7FAD": "Uniswap",
};
const appNameByAddr = dexProvidersContractAddress[transaction?.recipient || ""];
```

Changed it to this, is it ok?